tests/acceptance/bootstrap/SharingNgContext.php

```diff
@@ -691,6 +697,12 @@ public function updateResourceShare(string $user, TableNode  $body, string $perm
 			\json_encode($body),
 			$permissionID
 		);
+
+		if ($response->getStatusCode() === 200) {
+			$this->featureContext->shareNgAddToCreatedUserGroupShares($response);
```

Comment:
for update response, we should update the share instead of adding to the list. Check if `shareNgAddToCreatedUserGroupShares` method updates the share data properly
```suggestion
			$this->featureContext->shareNgAddToCreatedUserGroupShares($response);
```